hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FailuresSparkIT.java

```diff
@@ -129,15 +130,8 @@ public void testDeletedCellsInMemstoreDontHangSparkScans() throws Throwable {
                     }
                     con1.rollback();
 
-                    // force WW conflict to rollfoward (by deleting) the previous write
```

Comment:
The IT code to test a WW conflict is removed here. Should another test be added for this case, or it's not needed any more?

Reply:
The purpose was to rollforward data(removing a rolled back row). Code change for this PR already handle that. When the RollbackTxnFiter finds a rolled back row, it rolls it forward(delete it)